test/pools/integration/Cases.t.sol

```diff
@@ -211,6 +221,19 @@ contract TestCases is PoolsDeployer, Test {
             NAV_PER_SHARE.mulUint128(uint128(valuation.getQuote(APPROVED_SHARE_AMOUNT, USD.addr(), USDC_C2.addr())))
         );
         assertEq(m0.shareAmount, APPROVED_SHARE_AMOUNT);
+
+        // Simulate holding update response messages from CV after user claimed redeem
```

Comment:
Why do we need to simulate a holding update after a user claims? AFAIK, the holdings are already updated at this point. Would you want to move this section to line 206?

Reply:
Thanks for the ref 🙌🏻. Just a comment regarding this. The returning message will use subsidized with the pool funds. I'm not sure if we want permissionless calls spamming those messages.

An investor could do small investments that involves small claims to drain the subsidized funds for that pool. cc @mustermeiszer

Reply:
They are paid already by the used, the problem is the update holding message they will dispatch in the other side. But IIUC, @mustermeiszer said that this is not the case. Then I do not understand why they are there in the tests

Reply:
No, it currently definitely is. That is fine for now. We most likely will add a queu to the BSM and then it is batched as a summary at one point.